src/main/java/org/myrobotlab/service/interfaces/ServoControlPublisher.java

```diff
@@ -9,7 +10,7 @@
  */
 public interface ServoControlPublisher {
 
-  public ServoControl publishServoMoveTo(ServoControl sc);
+  public ServoMove publishServoMoveTo(Double pos);
```

Comment:
This should take a ServoMove object, not a double, right?

Reply:
yeah ... it could